src/main/kotlin/uk/gov/justice/digital/assessments/controllers/advice/ControllerAdvice.kt

```diff
@@ -145,7 +151,7 @@ class ControllerAdvice {
   @ResponseStatus(HttpStatus.FORBIDDEN)
   fun handle(e: UserIdIsMandatoryException): ResponseEntity<ErrorResponse?> {
     log.info("UserIdIsMandatoryException: {}", e.message)
-    return ResponseEntity(ErrorResponse(status = 401, developerMessage = e.message), HttpStatus.FORBIDDEN)
+    return ResponseEntity(ErrorResponse(status = 403, developerMessage = e.message), HttpStatus.FORBIDDEN)
   }
```

Comment:
this was incorrectly returning 401 unauthenticated but should be 403 forbidden as the status code